shared/src/db/database.ts

```diff
@@ -466,17 +466,19 @@ class database extends Dexie {
      * @returns
      */
     toIsoDateTime(date: number) {
-        return DateTime.fromMillis(date).toISO({
-            includeOffset: false,
-            suppressSeconds: true,
-        });
+        return DateTime.fromMillis(date)
+            .setLocale(navigator.language || "en-US")
+            .toISO({
+                includeOffset: false,
+                suppressSeconds: true,
+            });
     }
 
     /**
      * Convert a DateTime object to a numeric (UNIX) date
      */
     fromDateTime(date: DateTime) {
-        return date.toMillis();
+        return date.setLocale(navigator.language || "en-US").toMillis();
```

Comment:
not sure if the locale needs to be set when converting Luxon dates to Unix dates, but guess it does not do damage either.